spec/requests/api/subject_outcomes_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+describe "subject outcomes api" do
+  it "returns a unique list of outcomes for the given subject" do
+    subject = create(:subject, number: '1.234')
+    outcome,_ = create_pair(:outcome)
+    assessment = create_pair :direct_assessment,
+      subject: subject,
+      outcomes: [outcome]
```

Comment:
I like the style here for factories (name on line with method call, hash elements on their own lines). Adding parens makes it not as nice looking and confuses vim. The thoughtbot systle guide (each argument on its own line) would keep vim happy but this is a place where I like to stray.